mailpoet/views/woo_system_info.html

```diff
@@ -0,0 +1,38 @@
+<table class="wc_status_table widefat" cellspacing="0">
+  <thead>
+  <tr>
+    <th colspan="3" data-export-label="MailPoet"><h2><%= __("MailPoet") %></h2></th>
```

Comment:
The other plugins add a tooltip on the header. What do you think of doing the same?
<img width="1268" alt="Screenshot 2023-05-19 at 1 08 31 PM" src="https://github.com/mailpoet/mailpoet/assets/8002881/0c5f8072-7991-47dd-9563-62275ce894fc">

Reply:
19c4e5678